R/IonDb-methods.R

```diff
@@ -0,0 +1,79 @@
+#' @include IonDb.R
+
+#' @importFrom methods show
+#'
+#' @export
+setMethod("show", "IonDb", function(object) {
+    cat("class:", class(object), "\n")
```

Comment:
To reduce redundancy I would suggest to call here `callNextMethod()` and then only print the `IonDb` specific information (i.e. get the connection, extract the numbers of ions and have the `cat(" ion count:")`

Reply:
Thank you! I didn't know of `callNextMethod()`